smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -1182,9 +1182,8 @@ private poll() {
 	def host = getBridgeIP()
 	def uri = "/api/${state.username}/lights/"
 	log.debug "GET: $host$uri"
-	sendHubCommand(new physicalgraph.device.HubAction("""GET ${uri} HTTP/1.1
-HOST: ${host}
-""", physicalgraph.device.Protocol.LAN, selectedHue))
+	sendHubCommand(new physicalgraph.device.HubAction("GET ${uri} HTTP/1.1\r\n" +
```

Comment:
I think I'll leave this commit as-is for now as it has been tested.  I agree that using the alternate signature likely makes sense as a change to put in place down the road.